backend/src/db/types/User.ts

```diff
@@ -0,0 +1,29 @@
+import { Integer, Node } from 'neo4j-driver'
+
+export interface UserDbProperties {
+  allowEmbedIframes: boolean
+  createdAt: string
+  deleted: boolean
+  disabled: boolean
+  emailNotificationsChatMessage: boolean
+  emailNotificationsCommentOnObservedPost: boolean
+  emailNotificationsFollowingUsers: boolean
+  emailNotificationsGroupMemberJoined: boolean
+  emailNotificationsGroupMemberLeft: boolean
+  emailNotificationsGroupMemberRemoved: boolean
+  emailNotificationsGroupMemberRoleChanged: boolean
+  emailNotificationsMention: boolean
+  emailNotificationsPostInGroup: boolean
```

Comment:
I think most of those fields are optional - that also includes `deleted`?

```suggestion
  emailNotificationsChatMessage?: boolean
  emailNotificationsCommentOnObservedPost?: boolean
  emailNotificationsFollowingUsers?: boolean
  emailNotificationsGroupMemberJoined?: boolean
  emailNotificationsGroupMemberLeft?: boolean
  emailNotificationsGroupMemberRemoved?: boolean
  emailNotificationsGroupMemberRoleChanged?: boolean
  emailNotificationsMention?: boolean
  emailNotificationsPostInGroup?: boolean
```

Reply:
Right, and there were a lot if fields missing, that are not defined in neode. I already did this but somehow forgot to push